records/generator.go

```diff
@@ -253,10 +299,10 @@ func (rg *RecordGenerator) frameworkRecords(sj state.State, domain string, spec
 		host, port := f.HostPort()
 		if address, ok := hostToIP4(host); ok {
 			a := fname + "." + domain + "."
-			rg.insertRR(a, address, "A")
+			rg.insertRR(a, address, A)
 			if port != "" {
 				srv := net.JoinHostPort(a, port)
-				rg.insertRR("_framework._tcp."+a, srv, "SRV")
+				rg.insertRR("_framework._tcp."+a, srv, SRV)
```

Comment:
This is a little weird with srv, SRV -- I like above where we did address, maybe srvAddress?